src/libraries/System.Diagnostics.Process/tests/Helpers.cs

```diff
@@ -12,6 +12,8 @@ namespace System.Diagnostics.Tests
 {
     internal static class Helpers
     {
+        public const int PassingTestTimeoutMilliseconds = 60_000;
```

Comment:
seems a bit large, wouldn't 10s be sufficient?

Reply:
I wanted something conservative to cover slow ARM platforms. The test will not block and it will continue if finished sooner.